src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -893,7 +893,9 @@ public function testCollectDenormalizationErrors(?ClassMetadataFactory $classMet
 
             $this->fail();
         } catch (\Throwable $th) {
+            /** @var PartialDenormalizationException $th */
             $this->assertInstanceOf(PartialDenormalizationException::class, $th);
+            $this->assertNull($th->getExtraAttributesError());
```

Comment:
I don't think you need to test the content of the exception there, as other dedicated tests will cover it (same for the next occurrence)